app/main.py

```diff
@@ -1,6 +1,31 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
+
+
+def clear_cache() -> None:
+    cached_data.clear()
```

Comment:
There won't be need in this function if you'll just create a dict in `cache` function. It will create a new object every time we use a decorator for a new function